python_tests/test_python_frontend.py

```diff
@@ -1468,6 +1469,50 @@ def fusion_func(fd: FusionDefinition) :
             self.assertEqual(torch.real(inputs[0]), nvf_out[0])
             self.assertEqual(torch.imag(inputs[0]), nvf_out[1])
 
+    def test_pad(self):
```

Comment:
nitpick: Let's also try to test cache hit. I think current TRIE look up would just ignore the pad size during lookup and we'll hit the wrong kernel.

def fusion_func_pad1(fd: FusionDefinition):
  t0 = fd.from_pytorch(tensor0)
  t1 = fd.ops.pad(t0, [1, 1])
  fd.add_output(t1)

def fusion_func_pad2(fd: FusionDefinition):
  t0 = fd.from_pytorch(tensor0)
  t1 = fd.ops.pad(t0, [2, 2])
  fd.add_output(t1)